### PR TITLE
Local Instance Provider

### PR DESCRIPTION
**Pull request by arn3tt**

### CODE REVIEW

File: src/main/java/org/fogbowcloud/manager/core/instanceprovider/InstanceProvider.java [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+import org.fogbowcloud.manager.core.models.orders.ComputeOrder;
 import org.fogbowcloud.manager.core.models.orders.Order;
 import org.fogbowcloud.manager.core.models.orders.instances.OrderInstance;
 import org.fogbowcloud.manager.core.models.token.Token;
 
 public interface InstanceProvider {
 
 	/**
-	 * Request an Instance for an Order.
+	 * Requests an Instance for a ComputeOrder.
```
Comment (paulofelipefeitosa) [inline src/main/java/org/fogbowcloud/manager/core/instanceprovider/InstanceProvider.java]: This is a general implementation.

File: src/main/java/org/fogbowcloud/manager/core/instanceprovider/LocalInstanceProvider.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            throw new UnsupportedOperationException("Not implemented yet.");
+        } else {
+            throw new UnsupportedOperationException("Not implemented yet.");
+        }
+    }
+
+    private OrderInstance requestInstance(ComputeOrder order) throws RequestException {
+        String imageName = order.getImageName();
+        String id = computePlugin.requestInstance(order, imageName);
+        return new OrderInstance(id, InstanceState.ACTIVE);
```
Comment (paulofelipefeitosa) [inline src/main/java/org/fogbowcloud/manager/core/instanceprovider/LocalInstanceProvider.java]: The InstanceState should be inactive.

Comment (arn3tt) [reply src/main/java/org/fogbowcloud/manager/core/instanceprovider/LocalInstanceProvider.java]: Nice catch, I will fix this soon.

File: src/main/java/org/fogbowcloud/manager/core/instanceprovider/InstanceProvider.java (hunk 1 above)
Comment (arn3tt) [reply src/main/java/org/fogbowcloud/manager/core/instanceprovider/InstanceProvider.java]: Right, will fix this.

**Pull request merged**